frontend/src/pages/Play.jsx

```diff
@@ -46,15 +46,16 @@ export default function Play() {
   };
 
   function nextLevel() {
-    pickMovie(movieIdArray);
+    pickMovie(movieIdArray, count + 1);
```

Comment:
count is starting at 1 and you're adding one here: you're droping the eight first elements